### DISCUSSION TITLE
Migration from Baby+ App

### DISCUSSION THREAD

**Discussion by Nippey** [babybuddy/babybuddy#428] [General]
Hi,  
is there already a tool to migrate the JSON export of the Philips Baby+ app?

I think it is no big deal, but maybe someone already did the work.

Else, I'd be happy to provide the tool later on

BR Nippey

**Comment by cdubz** [comment]
The “backend admin” of Baby Buddy does a generic import feature that may be able to address this reasonably well.

**Comment by Nippey** [comment]
Thank you for the hint.
Some search and replace is sufficient.

Height / Head Circumference: Couldn't find these in the backend. Did I miss something?

BR Nippey

**Comment by cdubz** [comment]
Hm yeah we missed implementing that. I've created #421 to review this generally.
